File: WindowsApplication1/WindowsApplication1/Deportivo1.vb
Public Class Deportivo1
    Private D1 As Deportivo = New Deportivo()

    Private Sub btn_Click(sender As Object, e As EventArgs) Handles btn.Click
        D1.Pmarca = txtMarca.Text
        D1.Ptalla = CInt(txtTalla.Text)
        D1.Pdeporte = txtDeporte.Text
        txtMarca.Text = ""
        txtTalla.Text = ""
        txtDeporte.Text = ""
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        MsgBox("Marca: " + D1.Pmarca + vbNewLine + "Talla: " + CStr(D1.Ptalla) + vbNewLine + "Deporte: " + D1.Pdeporte)
    End Sub
End Class

File: HCSZapato/HCSZapato/Deportivo1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace HCSZapato
{
    public partial class Deportivo1 : Form
    {
        Deportivo D1 = new Deportivo();
        public Deportivo1()
        {
            InitializeComponent();
        }

        private void btn_Click(object sender, EventArgs e)
        {
            D1.setMarca(txtMarca.Text);
            D1.setTalla(Int32.Parse(txtTalla.Text));
            D1.setDeporte(txtDeporte.Text);
            txtMarca.Text = "";
            txtTalla.Text = "";
            txtDeporte.Text = "";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            txtMarca.Text = D1.getMarca();
            txtTalla.Text = D1.getTalla().ToString();
            txtDeporte.Text = D1.getDeporte();
        }
    }
}
